### PR TITLE
[EditorInspector] Fix ImportDock and ProjectExport capitalize

### PR DESCRIPTION
**Pull request by guilhermefelipecgs**
Fix https://github.com/godotengine/godot/pull/21216#issuecomment-414344421

### CODE REVIEW

File: editor/import_dock.cpp [hunk 1]
```diff
@@ -423,6 +423,7 @@ ImportDock::ImportDock() {
 	import_opts = memnew(EditorInspector);
 	add_child(import_opts);
 	import_opts->set_v_size_flags(SIZE_EXPAND_FILL);
+	import_opts->set_enable_capitalize_paths(bool(EDITOR_GET("interface/inspector/capitalize_properties")));
```
Comment (guilhermefelipecgs) [inline editor/import_dock.cpp]: CC @akien-mga Maybe we should rename this path "interface/inspector/capitalize_properties" to a more generic.

Comment (guilhermefelipecgs) [reply editor/import_dock.cpp]: Or should I use a different names for this?

Comment (akien-mga) [reply editor/import_dock.cpp]: I think it should default to true in EditorInspector.

Comment (guilhermefelipecgs) [reply editor/import_dock.cpp]: Changed.

File: editor/editor_inspector.cpp [hunk 2]
```diff
@@ -1536,7 +1536,7 @@ void EditorInspector::update_tree() {
 
 					Color c = sscolor;
 					c.a /= level;
-					section->setup(path_name, path_name, object, c, use_folding);
+					section->setup(acc_path, path_name, object, c, use_folding);
```
Comment (guilhermefelipecgs) [inline editor/editor_inspector.cpp]: For reference, I inverted `path_name` with `acc_name` to fix section names:
captura de tela de 2018-08-20 17-10-31

**Pull request merged**